EDEngineer/Resources/Data/entryData.json

```diff
@@ -1874,4 +1874,2476 @@
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Tungsten Carbide",
+    "ValueCr": null,
+    "BarterCost": 11,
+    "BarterValue": 6,
+    "FormattedName": "tungstencarbide",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": ["LAB", "PROC"],
+    "ContainerType": ["Industrial Locker (S)", "Industrial Locker (L)", "Research Locker (L)"]
+    },
+    {
+    "Name": "Viscoelastic Polymer",
+    "ValueCr": 600,
+    "BarterCost": 11,
+    "BarterValue": 6,
+    "FormattedName": "viscoelasticpolymer",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Chemicals",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Weapon Component",
+    "ValueCr": 1000,
+    "BarterCost": 18,
+    "BarterValue": 9,
+    "FormattedName": "weaponcomponent",
+    "Kind": "OdysseyComponent",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": "Tech",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": [""]
+    },
+    {
+    "Name": "Accident Logs",
+    "ValueCr": 4000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "accidentlogs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": ["LAB", "PROC", "PWR", "AGRI"],
+    "ContainerType": ["Agricultural Data Port", "Data Port", "Industrial Data Port", "Laboratory Data Port", "Power Data Port"]
+    },
+    {
+    "Name": "Air Quality Reports",
+    "ValueCr": 3000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "airqualityreports",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Industrial Data Port", "Power Data Port"]
+    },
+    {
+    "Name": "Atmospheric Data",
+    "ValueCr": 4000,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "atmosphericdata",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Laboratory Data Port"]
+    },
+    {
+    "Name": "Audio Logs",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "audiologs",
+    "Kind": "OdysseyData",
+    "OriginDetails": ["Planetary Settlement"],
+    "Group": ",
+    "SettlementType": [""],
+    "BuildingType": [""],
+    "ContainerType": ["Security Data Port"]
+    },
+    {
+    "Name": "AX Combat Logs",
+    "ValueCr": null,
+    "BarterCost": null,
+    "BarterValue": null,
+    "FormattedName": "axcombatlogs",
+    "Kind": "OdysseyData",

```

Comment:
I think the separation between OdysseyData / OdysseyItem / OdysseyComponent isn't useful - having them all appear in the same tab under different group (the expanders) would be more compact

They could all have the kind `"OdysseyIngredient"` and be separated by their group (`"Group": "Item"` for the items)

Reply:
I will fix this.

Reply:
Shoudl be fixed now.